utils/check-credentials.js

```diff
@@ -0,0 +1,26 @@
+const { askYesNoQuestion } = require('./readline');
+
+module.exports = async function checkCredentials(
+    accountId,
+    networkId,
+    keyStore
+) {
```

Comment:
this should only run if the networkId is either `testnet` or `default`. maybe add a check:

`if(networkId !== 'testnet' || networkId !== 'default') return;`

That way you break out of these checks early if, say, the networkId is mainnet.